Makefile.core.mk

```diff
@@ -86,6 +86,15 @@ export ISTIO_BIN=$(GOBIN)
 
 export ISTIO_OUT:=$(TARGET_OUT)
 export ISTIO_OUT_LINUX:=$(TARGET_OUT_LINUX)
+
+# LOCAL_OUT should include architecture where we are currently running versus the desired.
```

Comment:
I am a bit confused when LOCAL_OUT should be used and when ISTIO_OUT should be?

Reply:
Oh is local-out is for running things, we never explicitly output things there?

Reply:
Correct. LOCAL_OUT is where you want run built artifacts from (ie, those artifacts should be of the OS/ARCH of the container if being run in the container or the local environment if not running in the container). I would have preferred to use *HOST* for specifying where we are running, *LOCAL* for the container, but we already were using *LOCAL* (copying from *TARGET*) so hard to come up with a good name.